server/app/src/data_harvester.py

```diff
@@ -259,6 +210,58 @@ def copy_snapshots(self, datasets: List[StandardizedDataset]):
         # Delete the tmp folder
         os.system('rm -rf ' + tmp_folder)
 
+    @staticmethod
+    def id_short_trials(tmp_folder: str):
+        # Collect markers files corresponding to trials that are too short
+        trials_folder = os.path.join(tmp_folder, 'trials/')
+        trials_to_remove: List[str] = []
+        for root, dirs, files in os.walk(trials_folder):
+            for file in files:
+                local_filepath = os.path.join(root, file)
+                if file.endswith('.trc'):
+                    trc_file = nimble.biomechanics.OpenSimParser.loadTRC(local_filepath)
+                    if len(trc_file.timestamps) < MIN_TRIAL_LENGTH:
+                        trials_to_remove.append(local_filepath)
+                elif file.endswith('.c3d'):
+                    c3d_file = nimble.biomechanics.C3DLoader.loadC3D(local_filepath)
+                    if len(c3d_file.timestamps) < MIN_TRIAL_LENGTH:
+                        trials_to_remove.append(local_filepath)
+        return trials_to_remove
+
+    @staticmethod
+    def prep_og_skeleton(tmp_folder: str):
```

Comment:
While I find this name charming, I think for a professional codebase we should probably call this `prep_unscaled_skeleton` or something 😂